vsphere/datadog_checks/vsphere/vsphere.py

```diff
@@ -65,18 +65,13 @@
 
 
 # Time after which we reap the jobs that clog the queue
-# TODO: use it
```

Comment:
This is actually used

Reply:
I don't think so, `self.jobs_status` is always `{}` so we never do anything.